packages/server/src/tests/amp/ampEpicModels.ts

```diff
@@ -27,22 +34,21 @@ export interface AMPEpic {
 /**
  * Models for the data published by the epic tool
  */
-export interface AmpEpicTestVariant {
-    name: string;
-    heading?: string;
-    paragraphs: string[];
-    highlightedText?: string;
-    cta?: Cta;
-    tickerSettings?: TickerSettings;
-    showChoiceCards?: boolean;
-    defaultChoiceCardFrequency?: ContributionFrequency;
-}
+const ampEpicTestVariantSchema = z.object({
+    name: z.string(),
+    heading: z.string().optional(),
+    paragraphs: z.array(z.string()),
+    highlightedText: z.string().optional(),
+    cta: ctaSchema.optional(),
+    tickerSettings: tickerSettingsSchema.optional(),
+    showChoiceCards: z.boolean().optional(),
+    defaultChoiceCardFrequency: contributionFrequencySchema.optional(),
+});
 
-export interface AmpEpicTest {
-    name: string;
-    priority: number;
```

Comment:
How come we don't need all of these parameters in the new 'ampEpicTestSchema'?

Reply:
they come from `testSchema` which we can extend from